### PR TITLE
Add PHP versions to version guidance in README.md

### PR DESCRIPTION
**Pull request by swordbeta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,14 +75,15 @@ composer.phar update
...
-|---------|-------------|---------------------|--------------|---------------------|---------------------|-------|
-| 3.x     | EOL         | `guzzle/guzzle`     | `Guzzle`     | [v3][guzzle-3-repo] | [v3][guzzle-3-docs] | No    |
-| 4.x     | EOL         | `guzzlehttp/guzzle` | `GuzzleHttp` | N/A                 | N/A                 | No    |
-| 5.x     | Maintained  | `guzzlehttp/guzzle` | `GuzzleHttp` | [v5][guzzle-5-repo] | [v5][guzzle-5-docs] | No    |
-| 6.x     | Latest      | `guzzlehttp/guzzle` | `GuzzleHttp` | [v6][guzzle-6-repo] | [v6][guzzle-6-docs] | Yes   |
+| Version | Status     | Packagist           | Namespace    | Repo                | Docs                | PSR-7 | PHP Version |
+|---------|------------|---------------------|--------------|---------------------|---------------------|-------|-------------|
+| 3.x     | EOL        | `guzzle/guzzle`     | `Guzzle`     | [v3][guzzle-3-repo] | [v3][guzzle-3-docs] | No    | >= 5.3.3    |
+| 4.x     | EOL        | `guzzlehttp/guzzle` | `GuzzleHttp` | [v4][guzzle-4-repo] | N/A                 | No    | >= 5.4.0    |
```
Comment (sagikazarmark) [inline README.md]: If we use 5.5 two lines bellow, can we use 5.4 here?

**Comment by sagikazarmark** [comment]
Thank you for your contribution. Can you please take a look at my comment above? After that I am happy to merge it.

**Comment by swordbeta** [comment]
@sagikazarmark Fixed!

**Comment by sagikazarmark** [comment]
Awesome, thank you very much.

**Pull request merged**